dynamo-event-to-datacite-request/src/main/java/no/unit/nva/events/models/AwsEventBridgeEvent.java

```diff
@@ -39,12 +39,12 @@ public AwsEventBridgeEvent() {
     @JacocoGenerated
     @JsonProperty("region")
     public String getRegion() {
-        return region.getName();
+        return Optional.ofNullable(region).map(Regions::getName).orElse(null);
```

Comment:
Why not make return type `Optional<String>` ?
or is it totally fine that this is `null` for the caller?

Reply:
Basically all fields can be optional. I just wanted to provide the region directly instead of people having to parse it. Actually no one is going to use this value.

Reply:
But we do support it by using the `Jdk8Module` in `JsonUtils.objectMapper`.

Reply:
If we change the serializer or the serialization format then we should change what the getter returns. In principle I am not against a getter returning an Optional, but I am reluctant to the case where we need to serialize stuff because of the way Java handles Optionals in seriliazation. Especially since we serialize Optionals back to nulls and we do not distinguish between the cases of something not being set and something being set to not having a value.

Especially in this case what is an optional field and what is not, it is up to each handler and each use case, so in a way this class says, "I present you with all information that I have, deal with it."